packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -346,43 +353,43 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
     /* @conditional-compile-remove(rooms) */
     role
   } = props;
+
+  const getQueryOptions = (options: {
```

Comment:
move this helper function outside of the render function body (either the bottom of the file or its own file)